documentation/common/snip-prerequisites-openshift.adoc

```diff
@@ -1,4 +1,4 @@
-:context: enmasse
+:context: maas
```

Comment:
We're only going to want context defined in master.adoc unless it's when including a file we plan to reuse. Then we want it above the include statement. As this is a snippet, defining the context here will cause some big issues.

Reply:
Fixed for both snip files